manifests/z/Zheguisoft/zg-zsso/2.0.2/Zheguisoft.zg-zsso.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.zg-zsso
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: Shanghai Zhegui Software Co., Ltd.

```

```suggestion
Publisher: 上海折桂软件有限公司
```
This needs to match Control Panel for en-US devices.

Since the Publisher is corrupted I used Locale Emulator.